src/components/ChatHistoryButton/ChatHistoryButton.tsx

```diff
@@ -55,6 +55,8 @@ const ChatHistoryButton = () => {
 					showChatHistory();
 				}}
 				className="rcb-view-history-button"
+				role="button"
```

Comment:
@tjtanjin By the way, is there why we can't just use a button tag here instead of a div? I'm not sure what the decision is and that is why I added the role instead.

Reply:
Hey @deeola, adding of role here is correct. As for your question, we go with a `<div>` for greater flexibility and consistency in layout and styling. This helps to minimize instances where the chatbot appears to be different when added to various websites.